File: app/src/main/java/org/jay/launchstarterexample/tasks/delayinittask/DelayInitTaskA.java
package org.jay.launchstarterexample.tasks.delayinittask;


import org.jay.appstarter.MainTask;
import org.jay.launchstarterexample.utils.LogUtils;

public class DelayInitTaskA extends MainTask {

    @Override
    public void run() {
        // 模拟一些操作
        try {
            Thread.sleep(100);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
        LogUtils.i("DelayInitTaskA finished");
    }
}

File: app/src/main/java/org/jay/appstarterexample/tasks/delayinittask/DelayInitTaskA.kt
package org.jay.appstarterexample.tasks.delayinittask

import com.facebook.stetho.common.LogUtil
import org.jay.appstarter.MainTask

class DelayInitTaskA : MainTask() {
    override fun run() {
        // 模拟一些操作
        try {
            Thread.sleep(100)
        } catch (e: InterruptedException) {
            e.printStackTrace()
        }
        LogUtil.i("DelayInitTaskA finished")
    }
}